src/datastructures/selfbalancedbst/MyColorNode.java

```diff
@@ -1,22 +1,28 @@
 package datastructures.selfbalancedbst;
 
+import datastructures.arraylist.MyArrayList;
+import datastructures.linkedlist.MyLinkedList;
+
 public class MyColorNode {
     public enum Color{
         RED,
         BLACK
     }
-    private int value;
+    private double value;
+    private MyLinkedList elements;
```

Comment:
Please use generics for the elements.
I recommend you to install a Plugin called SonnarLint. That will warn you about using Raw classes when generics were expected.

Reply:
@JurgenPadilla fixed in next commit